src/sinks/prometheus.rs

```diff
@@ -14,11 +14,19 @@ use std::{collections::HashMap, net::SocketAddr, sync::Arc};
 use stream_cancel::{Trigger, Tripwire};
 use tracing::field;
 
-#[derive(Deserialize, Serialize, Debug)]
+#[derive(Deserialize, Serialize, Debug, Clone)]
 #[serde(deny_unknown_fields)]
 pub struct PrometheusSinkConfig {
     #[serde(default = "default_address")]
     pub address: SocketAddr,
+    #[serde(default = "default_buckets")]
+    pub buckets: Vec<f64>,
```

Comment:
It might be worth calling this `default_histogram_buckets` or something like that, just to be extra clear and so we don't have to rename it when we add the ability to override these defaults.